### ISSUE TITLE
test different resolution via the editor

### ISSUE THREAD

**Issue by wighawag** [mindjolt/starling-builder-editor#2]
How do I test the UI on different resolution ? it would be cool if the test and test game buttons would allow dynamic resizing of the window to see how it looks in multiple resolution

**Comment by yuhengh** [comment]
Hi, wighawag

The different resolution approach we recommend sits outside the scope of starling builder.
The reason we don't want to put it inside the editor is there are other ways to do that and we don't want to enforce it. For more information you can check out this article.
http://wiki.starling-framework.org/builder/multiple_resolution
It's implemented inside the demo project. You can test the demo project with different screen size on emulator.

Basically you can just set the canvas size to your design resolution in the editor.
With the property align/anchor code inside the game you will easily adapt to any resolution.

**Comment by ghost** [comment]
Closing this since it's implemented.
